packages/forms-web-app/__tests__/unit/routes/full-appeal/index.test.js

```diff
@@ -33,5 +34,6 @@ describe('routes/full-appeal/index', () => {
     expect(use).toHaveBeenCalledWith(decisionDateRouter);
     expect(use).toHaveBeenCalledWith(dateDecisionDueRouter);
     expect(use).toHaveBeenCalledWith(priorApprovalExistingHomeRouter);
+    expect(use).toHaveBeenCalledWith(useExistingServiceEnforcementNotice);
```

Comment:
line 25 should `expect(use.mock.calls.length).toBe(12)` as I added `useExistingServiceApplicationType` (line 31) in my merged PR

Reply:
That has already been changed and build is running.